lib/views/error-view.js

```diff
@@ -11,6 +9,7 @@ export default class ErrorView extends React.Component {
 
     retry: PropTypes.func,
     logout: PropTypes.func,
+    openDevTools: PropTypes.func,
```

Comment:
How is `openDevTools` used? I can't seem to find any instance in the codebase that passes this prop.

Reply:
Oh! It isn't 😅 

I briefly added an "open devtools" button to one of the offline views because you can't see the _actual_ network error anywhere else (the Error that `fetch` rejects with doesn't include it). So I had to drill `atom.openDevTools` everywhere. I decided against it to keep things more simple, but it looks like this propType missed the purge. I'll 🔥 .